src/index.js

```diff
@@ -31,16 +32,18 @@ const after = props => `
 }
 `
 
-const before = () => `
-content: '';
-position: absolute;
-top: 0;
-left: 0;
-width: 100%;
-z-index: 1;
-border-top-left-radius: 0.4em;
-border-top-right-radius: 0.4em;
-background-clip: padding-box;
+const before = props => `
+  content: '${props.topbarTitle | ''}';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 1;
+  border-top-left-radius: 0.4em;
+  border-top-right-radius: 0.4em;
+  background-clip: padding-box;
+  font-family: -apple-system, BlinkMacSystemFont, Helvetica, Arial, sans-serif, "Apple Color Emoji";
```

Comment:
Using the system font stack from https://css-tricks.com/snippets/css/system-font-stack/ but only the Mac ones.